css/components/footer.css

```diff
@@ -76,22 +76,3 @@
 .footer__contact:last-child {
   margin-bottom: 30px;
 }
-
-@media screen and (min-width: 1440px) {

```

Comment:
I put all the media queries in one file as this would allow for a better overview what happens in each media-query.

Reply:
I think I have to disagree on this one. 
If we base our css in components then we should also add the media-queries to the corresponding component. If we don't do it then we would face on a really big project the same issue as if we would only have one css file; the media-query css file would become way to big and unreadable.

Reply:
I think this is personal preference. For me I like to have one place where all my media queries are and where I can see which styles change with which media query. But your point is valid that it would be a pretty huge file after a while 😅

Reply:
like always: "it depends" ;)

Reply:
Exactly!